.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+        python-version: ['3.6','3.7','3.8']
+        experimental: [false]
+        include:
+          - python-version: '3.9-dev'
```

Comment:
nit: Could you change the order of this section to
```
- os
  python-version
  experimental
```
to match lines 16-18? Otherwise, the 3.9-dev build has a different name from the others (https://github.com/google/pytype/pull/743/checks), which is slightly confusing.